src/main/io/osd.c

```diff
@@ -628,6 +657,11 @@ void osdDrawElements(void)
         osdDrawSingleElement(OSD_HOME_DIR);
     }
 #endif // GPS
+#ifdef MAG
+    if (sensors(SENSOR_MAG) || sensors(SENSOR_GPSMAG)) {
```

Comment:
I would have it be available all of the time. This is in line with other uses of `attitude.values.yaw` where it's up to the user to decide if the data is good enough with the sensors that they've got.